### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by JonathanGrossmanITC** [AnnaLebed/Micro-blogging-project#1]
Hi Anna,

Great job so far! Here is some feedback.

1.  I'm reminding everyone to be sure you are creating a good flow / user experience with loaders and alerts. For instance, it's common to show a user a loader while information is loading onto the page or when the sends information in a post request. It's also common to show the user a success or error message after submitting a post request.

2. You do a good job of disabling button / changing style. I'm also reminding everyone to be sure you are creating a good flow / user experience with input checks and buttons. For instance, it's common to disable a button and give it an opaque style if no text is input or if the input is invalid and then toggle the button to enable and full color when the input is valid and submittable. You generally should validate inputs, too. For instance, here, you should check whether the input is of a certain length. Any other checks you can implement too.

3. When you incorporate loaders, alerts, and buttons, you can make a component for each and then reuse them throughout your code, and control them using props / state.

4. Are you using local storage yet?

**Comment by AnnaLebed** [comment]
No, local storage I'm not using 100% yet. Just saved the information
there....but, as I understood, I need also to get the information from the
local storage(because my tweets are not saved on the page)...just couldn't
find yet appropriate information about this.

вс, 10 мая 2020 г., 15:56 JonathanGrossmanITC <notifications@github.com>:

**Comment by AnnaLebed** [comment]
Hi Jonathan! How are you?

Thanks a lot for your feedback, it is very helpful and I really appreciate
it!

I've decided to start June cohort. So, see you soon )

Sincerely yours,
Anna Lebid

вс, 10 мая 2020 г. в 17:34, Anna Lebed <annettlebid@gmail.com>:

**Comment by JonathanGrossmanITC** [comment]
Thanks for letting me know. Good luck and see you soon.

Jon
